examples/gradle-v2/kotlin-multiplatform-example/gradle.properties

```diff
@@ -5,3 +5,9 @@ org.gradle.parallel=true
 
 org.jetbrains.dokka.experimental.gradle.pluginMode=V2Enabled
 org.jetbrains.dokka.experimental.gradle.pluginMode.noWarn=true
+
+# kotlin.native.enableKlibsCrossCompilation must be set to `true`
+# otherwise Kotlin can't generate a Klib for Coroutines in macosMain
+# when generating on Linux machines, resulting in 'Error class: unknown class'
+# for CoroutineScope appearing in the generated docs.
+kotlin.native.enableKlibsCrossCompilation=true
```

Comment:
```suggestion
# kotlin.native.enableKlibsCrossCompilation could be set to `true`
# to be able to generate documentation via Dokka for macosMain on Linux machines
# Otherwise, Dokka will produce 'Error class: unknown class'
# for CoroutineScope appearing in the generated docs because of absent klibs
kotlin.native.enableKlibsCrossCompilation=false
```

Would be nice not to introduce experimental flags into example projects if possible